bootstrap.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+curl -sSLO https://github.com/shyiko/ktlint/releases/download/0.29.0/ktlint &&
```

Comment:
Bootstrapping `ktlint` like this is problematic for a couple of reasons:

* It assumes a specific OS layout
* It potentially overrides other installs of `ktlint`

While it's only bootstrapping `ktlint` for now, it's safe enough to ignore this if one knows it's already installed.  If we have other items to bootstrap, this will need to be reworked.

Reply:
Ok. I'm just blinding copying the `brew`-independent install from the `ktlint` documentation; I think it would be valid to update our `install.md` document with required libraries, rather than have this as part of `bootstrap.sh`.